app/classifier/tasks/drawing/summary.cjsx

```diff
@@ -8,6 +10,13 @@ module.exports = React.createClass
     annotation: null
     expanded: false
 
+  getCorrectSingularOrPluralOfDrawingType: (type, number) ->
+    if number>1 then "#{type}s" else type
+
+  stripMarkdownFromLabel: (label) ->
+    label = label.replace /\!\[[^\]]*\]\([^)]*\)/g, ""
+    remark.process(label)
```

Comment:
Good point. Didn't think of that. Let's wait and see if that ever happens. Thanks for the merge.